pkg/device/device-util.go

```diff
@@ -217,24 +230,24 @@ func getAllPartsUsed(diskMetaName string, partitionName string) ([]PartUsed, err
 //
 // results in diskName arg as "sdc" and row as list of columns
 // [2,2097152B,9500469755903B,9500467658752B,ext4,5d8d56cb-e291-4dfd-81ac-fb664dd5ec75]
-func parsePartUsed(diskName string, row []string) (PartUsed, error) {
-	p := PartUsed{DiskName: diskName}
+func parsePartUsed(diskPath string, row []string) (PartUsed, error) {
+	p := PartUsed{DiskPath: diskPath}
 	if len(row) < 5 {
```

Comment:
convert to constant

Reply:
Havent changed this value, since I doubt whether this is 5 or 6. Since the no .of columns are 6 in case of used partition. Need to reconfirm, why 5 was used.

Reply:
we are strict on using the meta partition so that we don't pick any other disk by accident. the column will be 5 if we have fstype. it will be 4 without fstype. here we are enforcing that the metapartition does not have fstype

Reply:
Done.